src/gosling-brush/linear-brush-model.ts

```diff
@@ -84,7 +84,7 @@ export class LinearBrushModel {
             d3Drag: hgLibraries.d3Drag
         };
 
-        this.style = Object.assign(BRUSH_STYLE_DEFAULT, style);
+        this.style = Object.assign({}, BRUSH_STYLE_DEFAULT, style);
```

Comment:
To prevent from `BRUSH_STYLE_DEFAULT` being edited

Reply:
Ah good catch!